bitrise.yml

```diff
@@ -290,6 +294,32 @@ workflows:
         - text: Build status using latest Rust-Component
         - webhook_url: $WEBHOOK_SLACK_TOKEN_2
 
+  run_firefox_ui_tests_ios26:
```

Comment:
I picked out the tests that are known to pass on iOS 26 here to provide *some* protection against iOS 26 at the time the PRs are opened.